src/data/Endpoint.js

```diff
@@ -39,6 +39,7 @@ const endpoints = {
     DRIVER: `${laravelUrlPrefix}/${version}/customer/driver`,
     DRIVER_ROUTE: `${laravelUrlPrefix}/${version}/customer/driver-route`,
     JOB: `${laravelUrlPrefix}/${version}/customer/job/{0}`,
+    JOB_PRIVATE: `${laravelUrlPrefix}/${version}/customer/job/private`,
```

Comment:
bit confusing difference between JOB and JOB_PRIVATE @vinay3206

Reply:
@maartenhemmes  renamed to JOB_FROM_ROUTE